src/libZilliqa/Zilliqa.cpp

```diff
@@ -289,7 +289,9 @@ Zilliqa::Zilliqa(const PairOfKey& key, const Peer& peer, SyncType syncType,
       case SyncType::GUARD_DS_SYNC:
         LOG_GENERAL(INFO, "Sync as a ds guard node");
         m_ds.m_awaitingToSubmitNetworkInfoUpdate = true;
-        m_ds.StartSynchronization();
+        // m_ds.StartSynchronization();
```

Comment:
Remove this line and leave a comment on the logic of the following 2 lines. This is so that it is easier to understand the logic in the future

Reply:
Added.